index-Reversal.js

```diff
@@ -0,0 +1,67 @@
+//Guessing Game Project - Burlington Code Academy
+// started 2.5.20
+// due Monday, Feb. 10 by 9 a.m.
+// Written by Rob Mitchell
+// Role reversal - computer thinks of a number, human guesses
+// reversed from original index.js file 
+
+//set the standard input 
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+//Begins with computer asking to play a game
+function guessHuman() {
+    let guessCount = 1; //Sets initial count for guesses. Sets to one b/c first guess is outside count loop
+    //sets up user
+    console.log("Let's play a guessing game where you (human) \ntry to guess a number that I (computer) have chosen!")
+    //starts async function
+    async function start() {
+        let rightGuess = null;
+        //asks for an lower bound to the guessing range
+        let minRange = (await ask("Let's set a range for you to guess between. \nHow low can the number I pick be?")).trim();
+        //asks for the upper bound
+        let maxRange = (await ask("How high can the number be?"));
```

Comment:
When you set the minRange and maxRange, be sure to get a number from the input, otherwise what the user inputs will be assigned to maxRange as a string and will make your program behave in a weird way. When I input 1 and 100 for min and max ranges, respectively, and then I make a guess of 60 as my first guess, it tells me that my guess is out of the agreed upon range.